components/tabs/src/sync/record.rs

```diff
@@ -82,7 +82,10 @@ impl TabsRecord {
             .into_iter()
             .collect::<Result<_>>()?;
 
-        let ttl: u32 = payload.data["ttl"].as_u64().unwrap_or_default() as u32;
+        let ttl: u32 = match payload.data.get("ttl") {
+            Some(v) => v.as_u64().unwrap_or_default() as u32,
+            None => u32::default(),
```

Comment:
I just wanted to note that I have no idea if this is the appropriate value to use here, but followed what was already set there with the `unwrap_or_default()` I imagine a ttl of 0 is a special value that's ok to use?

Reply:
Yep, zero seems as good any to me.